test/catalog-compat/catcompatck/catcompatck

```diff
@@ -171,6 +171,26 @@ $ set schema={
   FROM real_time_src_no_stats
 EOF
 
+say "adding table with char column"
+testdrive --kafka-addr kafka:9092 --no-reset <<'EOF'
+# Prior to v0.9.1 char references essentially aliased `text`, despite being its
+# own entry in the catalog
+> CREATE TABLE str_col (a pg_catalog.char)
```

Comment:
could you please test with a CHAR(N) column here as well?

Reply:
fixed